deployments/multiple-ragapps/docker-compose.yml

```diff
@@ -0,0 +1,88 @@
+services:
+  traefik:
+    image: "traefik:v3.1"
+    ports:
+      - 80:80
+    labels:
+      - traefik.enable=true
+      - traefik.docker.network=ragapp-network
+      # Default username: admin, password: admin
+      - traefik.http.middlewares.admin-auth.basicauth.users=${USERNAME:-admin}:${HASHED_PASSWORD:-$$apr1$$3gG256rA$$BomHrC/uMhL807k/t7Wd6/}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    command:
+      - --providers.docker
+      - --providers.docker.exposedbydefault=false
+      - --entrypoints.http.address=:80
+      - --accesslog
+      - --log
+      - --api
+    networks:
+      - ragapp-network
+
+  ragapp-cs50:
+    build: ../..
+    image: ragapp/ragapp:latest
```

Comment:
didn't use build in my test, but was pulling `ragapp/ragapp:latest`

Reply:
Calling `docker-compose up --build` should do the trick - we should probably add it to the readme